cocos/platform/android/CCFileUtils-android.cpp

```diff
@@ -249,6 +268,12 @@ FileUtils::Status FileUtilsAndroid::getContents(const std::string& filename, Res
     } else {
         relativePath = fullPath;
     }
+    
+    if (obbfile)
+    {
+        if (obbfile->getFileData(relativePath, buffer))
+            return FileUtils::Status::OK;
```

Comment:
I'm not sure what you mean.
It will return OK if it is found in the OBB, otherwise it will try to find it in the APK (so it must not return if getFileData returns false).
Is that what you meant?
I can add a comment to make it clearer maybe?